packages/block-library/src/navigation/edit/placeholder/index.js

```diff
@@ -76,8 +76,7 @@ export default function NavigationPlaceholder( {
 
 						{ canUserCreateNavigationMenus && (
 							<Button
-								// TODO: Switch to `true` (40px size) if possible
-								__next40pxDefaultSize={ false }
+								__next40pxDefaultSize
```

Comment:
I haven't being able to identify this change visually on the site, because the main component, `NavigationPlaceholder` is exported, but is not used anywhere in entire repository, Ref - https://github.com/search?q=repo%3AWordPress%2Fgutenberg%20export%20default%20function%20NavigationPlaceholder(&type=code,

It is being used here - https://github.com/WordPress/gutenberg/blob/13af7fd1958ff5dcd7e28f6b7c7b7d280a39ae51/packages/block-library/src/navigation/edit/placeholder/index.js#L17, 

@mirka, Can you please verify this.

Thank You,

Reply:
Yes, it doesn't seem to be used anymore 👍

Reply:
Thanks for confirmation 👍.